paddle/fluid/framework/distributed_strategy.proto

```diff
@@ -22,61 +22,89 @@ enum Mode {
   HETER = 4; // support XPU and GPU computing server
 }
 
-message DistributedStrategy {
-  optional Mode mode = 1 [ default = COLLECTIVE ]; // just for serialization
-  // collective training strategy
-  optional bool amp = 2 [ default = false ];
-  optional int32 amp_loss_scaling = 3 [ default = 32768 ];
-  optional bool recompute = 4 [ default = false ];
-  repeated string recompute_checkpoints = 5;
-  optional bool localsgd = 6 [ default = false ];
-  optional int32 localsgd_k_step = 7 [ default = 4 ];
-  optional bool dgc = 8 [ default = false ];
-  optional bool hierachical_allreduce = 9 [ default = false ];
-  optional int32 hierachical_allreduce_inter_ranks = 10 [ default = 1 ];
-  optional int32 nccl_comm_num = 11 [ default = 1 ];
-  optional bool gradient_merge = 12 [ default = false ];
-  optional int32 gradient_merge_k_step = 13 [ default = 1 ];
-  optional bool sequential_execution = 14 [ default = false ];
-  optional bool enable_backward_optimizer_op_deps = 15 [ default = true ];
-  optional bool lars = 16 [ default = false ];
-  optional bool lamb = 17 [ default = false ];
-  optional bool fuse_elewise_add_act_ops = 18 [ default = false ];
-  optional bool fuse_bn_act_ops = 19 [ default = false ];
-  optional bool enable_auto_fusion = 20 [ default = false ];
-  optional bool fuse_relu_depthwise_conv = 21 [ default = false ];
-  optional bool enable_inplace = 22 [ default = false ];
-  optional bool fuse_all_reduce_ops = 23 [ default = false ];
-  optional int32 num_iteration_per_drop_scope = 24 [ default = 1 ];
-  optional bool sync_batch_norm = 25 [ default = false ];
-  optional bool fuse_all_optimizer_ops = 26 [ default = false ];
-  optional bool sync_nccl_allreduce = 27 [ default = true ];
-  optional bool fuse_broadcast_ops = 28 [ default = true ];
-  optional int32 num_threads = 29 [ default = 1 ];
-  optional int32 num_iteration_per_run = 30 [ default = 1 ];
+message RecomputeConfig { repeated string checkpoints = 1; }
+
+message AMPConfig {
+  optional int32 init_loss_scaling = 1 [ default = 32768 ];
```

Comment:
`init_loss_scaling` should be a float parameter, as is defined in https://github.com/PaddlePaddle/Paddle/blob/5e07db15e66710a63e46cc874756db44a9f89f78/python/paddle/fluid/contrib/mixed_precision/decorator.py#L232.

Reply:
fixed. thx